### ISSUE TITLE
Dynamic Retarget time based on angular distance

### ISSUE THREAD

**Issue by micahnyc** [OpenSpace/OpenSpace#3314]
Right now, when you retarget, the time is fixed. Presumably we know where the item you are targeting is, and can do a calculation so that the larger the angle, the longer the retargeting time

**Comment by WeirdRubberDuck** [comment]
Just a note that the the "linear" rotation ni the camera path system already does this type of computation

https://github.com/OpenSpace/OpenSpace/blob/62b2a8c8c1508bcd994724d9cf847c1f6b065e5c/src/navigation/path.cpp#L345

**Comment by WeirdRubberDuck** [comment]
> Should we close this issue then?

No, the feature still has to be implemented for the retargeting when switching focus nodes (the computation only exists for the camera motions as of now)
